chia/rpc/wallet_rpc_api.py

```diff
@@ -1334,6 +1335,46 @@ async def verify_signature(self, request) -> EndpointResult:
         else:
             return {"isValid": False, "error": "Signature is invalid."}
 
+    async def check_ids(self, request) -> EndpointResult:
+        """
+        Check if a list of IDs is owned by the current wallet.
+        :param request: {"ids":["ID1", "ID2", ...], "type":"COIN/NFT/DID"}
```

Comment:
wouldn't it be easier to provide a list of `(coin_id, coin_type)` vs doing multiple requests depending on the coin type?

Reply:
I agree that being able to query a mix of coin types in a single request would be a nice.

Do we even need to specify the type if we can infer the coin type by the bech32m prefix? `nft1...abcxyz` for NFT, `did:chia:1...def123` for DID, and anything else could be looked up as a coin. Or is there a reason to have the type specified by the caller?

Reply:
The question is if we want to support hex string for NFT and DID. Also, if the type is a known thing, providing the type will speed up the query.